controllers/dummy_controller.go

```diff
@@ -0,0 +1,6 @@
+package controllers
```

Comment:
interesting...

Reply:
Not sure if there is a better way to do it. I guess the sdk just assumes you don't love around controllers after the generation. I also think putting the controller back and keeping the supplements in the package where they are currently would make sense. That way the api is separated from the business logic. But that maybe a future improvement, for now I wanted to make the new package structure for the existing one.

Reply:
will raise the Question to the operator folks to see there ideas (maybe something we didn't configure 100%)

Reply:
nothing to do here for now, added a additional commit. Resolving to better see what action items are still open.